config.yml

```diff
@@ -431,10 +431,25 @@ nodes:
     fields:
       - name: new_name
         type: node
+        comment: |
+          Represents a GlobalVariableReadNode.
+
+              alias $foo $bar
+                    ^^^^
       - name: old_name
         type: node
+        comment: |
+          Represents a GlobalVariableReadNode.
```

Comment:
We also need more information here:

* What is allowed to be aliased?
* Can there be anything other than global variables in this position?